app/src/main/java/com/iitp/njack/iitp_connect/core/youtube/PlaylistRepository.java

```diff
@@ -0,0 +1,139 @@
+            mService = new com.google.api.services.youtube.YouTube.Builder(
+                transport, jsonFactory, credential)
+                .setApplicationName("IITP-Connect")
+                .build();
+            youtubePlaylists.add(new YoutubePlaylist("http1"));
+        }
+
+        @Override
+        protected List<YoutubePlaylist> doInBackground(Void... params) {
+            try {
+                return getDataFromApi();
+            } catch (UserRecoverableAuthIOException e) {
+                ((Activity) context).startActivityForResult(e.getIntent(), 1);
+                return null;
+            } catch (Exception e) {
+                e.printStackTrace();
+                Log.e("error", "error occured connect");
+                youtubePlaylists.add(new YoutubePlaylist("error occured"));
+                cancel(true);
+                return null;
+            }
+        }
+
+        private List<YoutubePlaylist> getDataFromApi() throws IOException {
+            List<YoutubePlaylist> channelInfo = new ArrayList<>();
+            ChannelListResponse result = mService.channels().list("snippet,contentDetails,statistics")
+                .setForUsername("GoogleDevelopers")
+                .execute();
+            youtubePlaylists.add(new YoutubePlaylist("http2"));
+
+            List<Channel> channels = result.getItems();
+            if (channels != null) {
+                Channel channel = channels.get(0);
+
+                HashMap<String, String> parameters = new HashMap<>();
+                parameters.put("part", "snippet,contentDetails");
+                parameters.put("channelId", channel.getId());
+                parameters.put("maxResults", "25");
+                YouTube.Playlists.List playlistsListByChannelIdRequest = mService.playlists().list(parameters.get("part"));
+
+                if (parameters.containsKey("channelId") && !parameters.get("channelId").equals("")) {
+                    playlistsListByChannelIdRequest.setChannelId(parameters.get("channelId"));
+                }
+                if (parameters.containsKey("maxResults")) {
+                    playlistsListByChannelIdRequest.setMaxResults(Long.parseLong(parameters.get("maxResults")));
+                }
+
+                PlaylistListResponse playlistListResponse = playlistsListByChannelIdRequest.execute();
+                List<Playlist> playlist_list = new ArrayList<>();
+                playlist_list = playlistListResponse.getItems();
+
+                if (playlist_list != null) {
+                    for (Playlist playlist1 : playlist_list) {
+                        channelInfo.add(new YoutubePlaylist(playlist1.getId()));
+                        youtubePlaylists.add(new YoutubePlaylist(playlist1.getId()));
+                    }
+                }
+            }
+            youtubePlaylists.add(new YoutubePlaylist("hkhkkkhh"));
+            return channelInfo;
+        }
+
+        @Override
+        protected void onPreExecute() {
```

Comment:
Remove unnecessary code

Reply:
If we need it in the future, we will override it then. Remove them right now.